PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,10 @@
+Please try to adhere to the guidelines below as much as possible when submitting your pull request.
+
+- [ ] Please add `[WIP]` to the beginning of your pull request title if you are still working on it or it is not ready for review.
```

Comment:
this is a much better way of putting it, thanks. Is the info about `[MRG+x]` unnecessary? I found that difficult to describe.